botorch/acquisition/multi_objective/monte_carlo.py

```diff
@@ -750,3 +758,108 @@ def forward(self, X: Tensor) -> Tensor:
         samples = self._get_f_X_samples(posterior=posterior, q_in=q_in)
         # Add previous nehvi from pending points.
         return self._compute_qehvi(samples=samples, X=X) + self._prev_nehvi
+
+
+class MOMF(qExpectedHypervolumeImprovement):
```

Comment:
Let's make a separate file called multi_fidelity.py for this acquisition function

Reply:
I did this. Although I have not made any abstract class like in the other files. Would that be necessary too?